### ISSUE TITLE
Migrate to JavaPackager

### ISSUE THREAD

**Issue by wetneb** [OpenRefine/OpenRefine#2107]
We currently use Launch4j and Appbundler to package OpenRefine for Windows and Mac.
Appbundler is no longer supported by Oracle and they recommend JavaPackager instead. JavaPackager handles a lot of distribution formats and is also integrated with Maven via this plugin:
https://github.com/fvarrui/JavaPackager

We should consider migrating to that.

**Comment by thadguidry** [comment]
JavaPackager is now incubating in OpenJDK 14 via [JEP 343](https://openjdk.java.net/jeps/343).

**Comment by wetneb** [comment]
Interesting… I didn't realize JavaPackager had only been available in Java 8 too. Citing the specs for jpackager, the OpenJDK equivalent:

> There will be no support for cross compilation. For example, in order to create Windows packages one must run the tool on Windows. The packaging tool will depend upon platform-specific tools.

That does not sound great to me! I really like it that I can generate Windows packaging from Linux in the current setup :-/

**Comment by thadguidry** [comment]
Yeah, Mark also gives some good comments if you look into the [JEP 343 issue on the mailing list](https://mail.openjdk.java.net/pipermail/core-libs-dev/2019-June/060701.html) it seems there still is quite a bit of work to do on it as well.

**Comment by wetneb** [comment]
Actually jpackage and JavaPackager seem to be two distinct, unrelated things, as far as I can tell.
